geth/api/backend.go

```diff
@@ -236,3 +236,10 @@ func (b *StatusBackend) ConnectionChange(state ConnectionState) {
 	// logic of handling state changes here
 	// restart node? force peers reconnect? etc
 }
+
+// AppStateChange handles app state changes (background/foreground).
+func (b *StatusBackend) AppStateChange(state AppState) {
+	log.Info("App State changed: %s", state)
+
+	// TODO: put node in low-power mode
```

Comment:
Always, not depending on state? Also here the comment of the method could provide more information on what will happen based on the passed argument.

Reply:
ah, yeah, I'll fix the comment

Reply:
fixed